visualizer_user.c

```diff
@@ -140,7 +179,7 @@ void update_user_visualizer_state(visualizer_state_t* state) {
     }
     else {
         state->target_lcd_color = LCD_COLOR(0x40, 0xB0, 0xFF);
-        state->layer_text = "Default";
+        state->layer_text = "  :D :D :D :D :D :D";
```

Comment:
I think the default text serves better as a starting point for customization. So I would prefer that the text is left alone.